conf/sahmri.config

```diff
@@ -0,0 +1,29 @@
+params {
+  config_profile_description = 'South Australian Health and Medical Research Institute (SAHMRI) HPC cluster profile.'
+  config_profile_contact     = 'Nathan Watson-Haigh (nathan.watson-haigh@sahmri.com)'
+  config_profile_url         = "https://sahmri.org.au"
+  max_memory      = 375.GB
+  max_cpus        = 32
+  max_time        = 14.d
+  igenomes_base   = '/cancer/storage/shared/igenomes/'
+}
+process {
+  executor        = 'slurm'
+  queue           = 'sahmri_prod_hpc,sahmri_cancer_hpc'
+  maxRetries      = 2
+
+  cpus            = { check_max( 2 * task.attempt, 'cpus') }
+  memory          = { check_max( 1.GB * task.attempt, 'memory') }
+  time            = { check_max( 10.m * task.attempt, 'time') }
```

Comment:
Is there a purpose for setting this here? All nf-ore pipelines define default settings for these on a pipeline-basis so shouldn't be necessary. Or is the purpose here to support non-nf-core pipelines as well?

Reply:
The intention is to also use this for non nf-core pipelines too.